services/ows_refactored/vegetation/ows_fc_summary_annual_cfg.py

```diff
@@ -349,14 +349,12 @@
 ]
 
 layer = {
-    "title": "DE Africa Fractional Cover annual summary",
+    "title": "Fractional Cover annual summary",
     "name": "fc_ls_summary_annual",
-    "abstract": """Fractional Cover 30 m Percentiles (Landsat, Annual)
-Data is only visible at higher resolutions; when zoomed-out the available area will be displayed as a shaded region.
-
+    "abstract": """
 Fractional cover provides information about the the proportions of green vegetation, non-green vegetation (including deciduous trees during autumn, dry grass, etc.), and bare areas for every 30 m x 30 m ground footprint. Fractional cover provides insight into how areas of dry vegetation and/or bare soil and green vegetation are changing over time. The percentile summaries are designed to make it easier to analyse and interpret fractional cover. Percentiles provide an indicator of where an observation sits, relative to the rest of the observations for the pixel. For example, the 90th percentile is the value below which 90% of the observations fall. The fractional cover algorithm was developed by the Joint Remote Sensing Research Program.
 
-This contains a three band combination of the 50th Percentile for green vegetation, non green vegetation and bare soil observations acquired in each full calendar year (1st of January - 31st December) from 1984 to the most recent full calendar year.
+This product contains a three band combination of the 10th, 50th and 90th Percentile for green vegetation, non green vegetation and bare soil observations acquired in each full calendar year (1st of January - 31st December) from 1984 to the most recent full calendar year.
```

Comment:
This product contains the 10th, 50th, and 90th percentiles estimated independently for the green vegetation, non-green vegetation, and bare fractions observed in each calendar year (1st of January - 31st December) from 1984 to the most recent full calendar year.